docs/features/kubernetes/authentication.md

```diff
@@ -0,0 +1,35 @@
+---
+id: kubernetes-auth
+title: Kubernetes Authentication
+description: Authentication in Kubernetes plugin
+---
+
+The authentication process in kubernetes is basically separate from backstage auth, the
+providers are configured so your kubernetes plugin can locate and access the clusters you
+have access to, the providers currently available are categorized in server side auth and
+client side auth.
+
+## Server Side Providers
+
+These providers authenticate your _application_ with the cluster, meaning anyone that is
+logged in into your backstage app will be granted the same access to Kubernetes objects.
+The providers available as server side are:
+
+- `aws`
+- `azure`
+- `googleServiceAccount`
+- `localKubectlProxy`
+- `serviceAccount`
+
+## Client Side Providers
+
+These providers authenticate your user with the cluster, each user will be requested for
```

Comment:
italics to match the above section, break up a run-on sentence, and avoid the word 'request' which (for me at least) brings to mind HTTP requests
```suggestion
These providers authenticate your _user_ with the cluster. Each Backstage user will be prompted for
```